README.md

```diff
@@ -1,27 +1,20 @@
-# Kolla-G5K
+# Ham
 
 ## Synopsis
+
 This script deploys an OpenStack on Grid'5000 using
-[Kolla](https://wiki.openstack.org/wiki/Kolla) and allows easy customization of the system (change in the default topology, configuration parameters) and reproductible experiments.
+[Kolla](https://wiki.openstack.org/wiki/Kolla) and allows easy
+customization of the system (change in the default topology,
+configuration parameters) and reproductible experiments.
 
 ## Installation
-To install Kolla-G5K, first connect to the Grid'5000 frontend of your choice :
-and clone the repository
-
-```
-git clone https://github.com/BeyondTheClouds/kolla-g5k
-```
 
-Then install the dependencies:
+To install Ham, first connect to the Grid'5000 frontend of your
+choice and run.
 
 ```
-cd kolla-g5k
-pip install -r requirements.txt --user
+pip install git+git://github.com/BeyondTheClouds/ham@master#egg=ham
```

Comment:
Ok I briefly read some documentation and made some quick tests. On this particular point, I think the current state is ok.  

* There is no good reason to run `python ham/ham.py` in the source directory.  One should go with `python -m ham/ham.py` or install locally the package through `pip install -e  .` and use `ham <subcommand>`. 

* Pushing to Pypi the 1.0.0 version for Newton would be awesome :)